mtp_cashbook/assets-src/stylesheets/elements/_form-validation.scss

```diff
@@ -0,0 +1,106 @@
+$gutter: 30px;
+$gutter-half: $gutter/2;
+$error-colour: #AF1324;
+
+// Form validation
+// ==========================================================================
+
+// Using the classname .error as it's shorter than .validation and easier to type!
```

Comment:
Valid. Admittedly this is taken from the `govuk-elements` until it's added to `mojular`. 

Will remove that line though.